guessit/api.py

```diff
@@ -146,5 +150,86 @@ def properties(self, options=None):
             ordered = self.rebulk.customize_properties(ordered)
         return ordered
 
+    @staticmethod
+    def load_config(options):
+        """
+        Load configuration from configuration file, if defined.
+        :param options:
+        :type options:
+        :return:
+        :rtype:
+        """
+        config_file_options = None
+        if options.get('config'):
+            config_file = options.get('config')
+            if config_file and config_file in ['0', 'no', 'false', 'disabled']:
+                return options
+        else:
+            home_directory = os.path.expanduser("~")
+            config_file = GuessItApi.get_json_config_file(home_directory)
+            if not config_file:
+                config_file = GuessItApi.get_yml_config_file(home_directory)
+        if config_file:
+            if os.path.exists(config_file):
+                if config_file.endswith('.json'):
+                    with open(config_file) as config_file_data:
+                        config_file_options = json.load(config_file_data)
+                elif config_file.endswith('.yaml') or config_file.endswith('.yml'):
+                    try:
+                        import yaml
+                        with open(config_file) as config_file_data:
+                            config_file_options = yaml.load(config_file_data)
+                    except ImportError:  # pragma: no cover
+                        print('PyYAML is not installed. Options file "%s" will be ignored ...' % (config_file,),
+                              file=sys.stderr)
+                else:
+                    print('Configuration file "%s" has no supported extension. It should ends with json, yaml or yml. '
+                          'Using default options...' % (config_file,), file=sys.stderr)
+            else:
+                print('Configuration file "%s" was not found. Using default options...' % (config_file,),
+                      file=sys.stderr)
+        if config_file_options:
+            options.update(config_file_options)
```

Comment:
configuration file options will erase options given a command line ... We need to update only options that were not given initially.